a3p-integration/proposals/g:4-ymax-alpha3/test/wf.test.js

```diff
@@ -0,0 +1,75 @@
+// @ts-check
+import '@endo/init/legacy.js'; // axios compat
```

Comment:
master now has cosmjs that doesn't import axios.


```suggestion
import '@endo/init';
```

non-blocking. and actually we'd probably rather have this for tests,
```suggestion
import '@endo/init/debug.js';
```

Reply:
I'd like to address these last few comments in a follow-on PR.

Reply:
so much for that idea: [ci bot frowned on me](#issuecomment-3184725302)